aait/backend/group-2B/Application/DTOs/PostDtos/GeneralPostDto.cs

```diff
@@ -0,0 +1,13 @@
+
+using SocialSync.Application.DTOs.Common;
+
+namespace SocialSync.Application.DTOs.PostDtos;
+
+public class GeneralPostDto : BaseDto
+{
+    public int UserId { get; set; }
+    public string? Content { get; set; }
```

Comment:
Should "Content" be nullable? since it is the main component for creating the post, it should not be nullable.

Reply:
That's right. Post content can not be nullable. It is fixed now.